lib/widgets/sort_posts_picker.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:lemmy_api_client/lemmy_api_client.dart';
+
+import 'bottom_modal.dart';
+
+class SortPostsPicker extends HookWidget {
```

Comment:
Consider using a different name since it also handles switching between tile/card view